src/conversion.rs

```diff
@@ -288,7 +288,14 @@ where
 
     fn try_from(geometry: Geometry<T>) -> Result<Self, Self::Error> {
         Ok(match geometry {
-            Geometry::Point(g) => geo_types::Geometry::Point(g.try_into()?),
+            Geometry::Point(g) => {
+                // Special case as `geo::Point` can't be empty
+                if g.0.is_some() {
```

Comment:
Very minor thing, but we should be able to write this in an infallible way, like:
```
 if let Some(c) = g.0 {
    geo_types::Geometry::Point(geo_types::Point(c.into()))
```

Reply:
I wanted to allow any other error handling / extensions we might add in the future here.